pkg/controller/controller.go

```diff
@@ -275,10 +275,6 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 	var privKey []byte
 	var apicCert []byte
 
-	if gitCommit != "" {
```

Comment:
Is there any way in the logs developer can see the version / commit id after removing this ?

Reply:
The (equivalent)code has been moved to cmd/controller/main.go